DSCResources/MSFT_NetworkTeam/MSFT_NetworkTeam.psm1

```diff
@@ -0,0 +1,243 @@
+﻿# Fallback message strings in en-US
+DATA localizedData
+{
+    # same as culture = "en-US"
+ConvertFrom-StringData @'    
+    getTeamInfo=Getting network team information for {0}.
+    foundTeam=Found a network team with name {0}.
+    teamMembersExist=Members in the network team {0} exist as per the configuration.
+    teamNotFound=Network team with name {0} not found.
+    lbAlgoDifferent=Load Balancing Algo is different from the requested {0} algo.
+    teamingModeDifferent=Teaming mode is different from the requested {0} mode.
+    modifyTeam=Modifying the network team named {0}.
+    membersDifferent=Members within the team named {0} are different from that requested in the configuration.
+    removingMembers=Removing members {0} not specified in the configuration.
+    addingMembers=Adding members {0} that are not a part of the team configuration.
+    createTeam=Creating a network team with the name {0}.
+    removeTeam=Removing a network team with the name {0}.
+    teamExistsNoAction=Network team with name {0} exists. No action needed.
+    teamExistsWithDifferentConfig=Network team with name {0} exists but with different configuration. This will be modified.
+    teamDoesNotExistShouldCreate=Network team with name {0} does not exist. It will be created.
+    teamExistsShouldRemove=Network team with name {0} exists. It will be removed.
+    teamDoesNotExistNoAction=Network team with name {0} does not exist. No action needed.
+    waitingForTeam=Waiting for network team status to change to up.
+    createdNetTeam=Network Team was created successfully.
+    failedToCreateTeam=Network team with specific configuration failed to changed to up state within timeout period of 120 seconds.
+'@
+}
+
+if (Test-Path "${PSScriptRoot}\${PSUICulture}")
```

Comment:
I'm in two minds here - I like the idea of every culture being in it's own file (including en-us). But I took a look at what was done in the PSDesiredStateConfiguration Resource Module in WMF5.0 and the localization is performed like this:
```powershell
data LocalizedData
{
    # culture="en-US"
    ConvertFrom-StringData -StringData @'
    CheckSumFileExists = File '{0}' already exists. Please specify -Force parameter to overwrite existing checksum files.
    CreateChecksumFile = Create checksum file '{0}'
...
'@
}

Import-LocalizedData  -BindingVariable LocalizedData -FileName PSDesiredStateConfiguration.Resource.psd1 -ErrorAction SilentlyContinue
```
and the en-us culture is included in the base file. I think keeping the en-us in the base file is probably the way to go to just so there is at least something to easily refer back to in the core file (without having to jump back to another file to see what a particular message says).

I would say that the Import-LocalizedData needs to be split over multiple lines to come over the 100 char limit:
```powershell
Import-LocalizedData  `
   -BindingVariable LocalizedData `
   -FileName PSDesiredStateConfiguration.Resource.psd1 `
   -ErrorAction SilentlyContinue
```